jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -690,9 +704,13 @@ class DeploymentFlow extends React.Component {
   */
   _generateExpertBudgetSection() {
     const status = this._getSectionStatus('expert-budget');
+    const state = this.state;
     if (!status.visible) {
       return;
     }
+    const estimate = parseInt(state.ddEntity.toEntity().price, 10);
```

Comment:
Should we parse the int in the model?

Reply:
yeah that's a much better idea...I originally left it like this because it's used as strings elsewhere...but this makes it much easier to work with.

done